baybe/exceptions.py

```diff
@@ -1,6 +1,15 @@
-"""Custom exceptions."""
+"""Custom exceptions and warnings."""
 
 
+##### Warnings #####
+class NonPredictiveRecommenderWarning(UserWarning):
```

Comment:
`RedundantObjectWarning`? Like "you are providing an object for a recommendation process that is redundant. This can be the case in any of the following cases: ...."

Reply:
I'm not exactly sure about the use of the word "redundant" in English, tbh. If it's exactly like the German use to express that something is "überflüssig da bereits mehrfach vorhanden", then I think it's not what we want here. If it can be also interpreted like "unused", then yes. But perhaps then `UnusedObjectWarning` is even more explicit?

Reply:
renamed to `UnusedObjectWarning`, resolving

Reply:
nice, thx